FreeRTOS/FreeRTOS
# Set ETHERNET_IRQn priority in main_networking.c

Description
-----------
This allows the qemu-mps2 demo to use DHCP which requires ethernet mac functionality prior to when vApplicationIPNetworkEventHook or vApplicationIPNetworkEventHook_Multi are called.

Test Steps
-----------
Attempting to use the mps2-an385 TCP demo fails every time with DHCP due to this bug.

To validate the fix, set ipconfigUSE_DHCP to 1 and run the mps2 demo on a network with DHCP.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
